src/messages.tsx

```diff
@@ -0,0 +1,20 @@
+export const NO_CONTEXTS_SAVED = 'No contexts saved yet, run asyncapi --help to know more';
+
+export const CONTEXT_NOT_FOUND = 'This context key does not exists.';
+
+export const KEY_NOT_FOUND = 'Key not found';
+
+export const DELETING_CURRENT_CONTEXT = 'You are tyring to delete a context that is currently in use.';
+
+export const MISSING_CURRENT_CONTEXT = 'No context is set as current, please set a current context.';
```

Comment:
I don't think `MissingCurrentContextError` is ever used

Reply:
`MissingCurrentContextError` is used here https://github.com/asyncapi/cli/blob/2f69e066f00c1dcb8336b1f1f2d8aac09d23884b/src/hooks/context/hooks.tsx#L27

Reply:
oh, but such message should never be printed to the user. When user did not pass `file` or `context` option, and also did not specify a default context, we should look for default files in CWD, and if they are missing, we should print a nice message that explains possible options

Reply:
can be fixed with https://github.com/asyncapi/cli/issues/35 though

Reply:
Yeah I will be fixing this just after this PR.